main.go

```diff
@@ -19,253 +19,98 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
+	"github.com/warrensbox/terraform-switcher/lib/param_parsing"
 	"os"
 	"path/filepath"
 	"strings"
 
 	semver "github.com/hashicorp/go-version"
-	"github.com/hashicorp/hcl2/gohcl"
-	"github.com/hashicorp/hcl2/hclparse"
-	"github.com/hashicorp/terraform-config-inspect/tfconfig"
-	"github.com/mitchellh/go-homedir"
-
 	"github.com/manifoldco/promptui"
-	"github.com/pborman/getopt"
-	"github.com/spf13/viper"
-
 	lib "github.com/warrensbox/terraform-switcher/lib"
 )
 
 const (
-	defaultMirror = "https://releases.hashicorp.com/terraform"
-	defaultLatest = ""
-	tfvFilename   = ".terraform-version"
-	rcFilename    = ".tfswitchrc"
-	tomlFilename  = ".tfswitch.toml"
-	tgHclFilename = "terragrunt.hcl"
+	tfvFilename = ".terraform-version"
```

Comment:
The same filename is declared at `lib/param_parsing/terraform_version.go` — can we squash the duplicate please?

Reply:
Done